GetARock/GetARock/Screen/AddGathering/AddGatheringViewController.swift

```diff
@@ -9,9 +9,107 @@ import UIKit
 
 class AddGatheringViewController: UIViewController {
 
+    // MARK: - View
+
+    @IBOutlet weak var titleTextField: UITextField!
+    @IBOutlet weak var hostBandNameLabel: UILabel!
+    @IBOutlet weak var introductionTextView: UITextView!
+    @IBOutlet weak var scrollView: UIScrollView!
+
+    private let placeHolderLabel: UILabel = {
+        $0.text = "내용을 입력하세요"
+        $0.textColor = .lightGray
+        $0.sizeToFit()
+        $0.font = .preferredFont(forTextStyle: .subheadline)
+        return $0
+    }(UILabel())
+
+    // MARK: - Life Cycle
+
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        attribute()
+        setDelegate()
+        setupLayout()
+    }
+
+    // MARK: - Method
+
+    @IBAction func cancelButtonAction(_ sender: UIBarButtonItem) {
+        dismiss(animated: true)
+    }
+
+    @IBAction func scrollViewTapRecognizer(_ sender: UITapGestureRecognizer) {
+        view.endEditing(true)
+    }
+
+    private func attribute() {
+        setupNavigationBar()
+        hostBandNameLabel.text = "블랙로즈" // 추후 유저디폴트 사용 예정
+        titleTextField.becomeFirstResponder()
+        introductionTextView.delegate = self
+        getKeyboardNotification()
+    }
+
+    private func setDelegate() {
+        introductionTextView.delegate = self
+    }
 
+    private func setupNavigationBar() {
+        navigationController?.navigationBar.shadowImage = UIImage()
     }
 
+    private func setupLayout() {
+        view.addSubview(placeHolderLabel)
+        placeHolderLabel.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            placeHolderLabel.topAnchor.constraint(equalTo: introductionTextView.topAnchor, constant: 8),
+            placeHolderLabel.leadingAnchor.constraint(equalTo: introductionTextView.leadingAnchor, constant: 4)
+        ])
+    }
+}
+
+// MARK: - UITextViewDelegate
+
+extension AddGatheringViewController: UITextViewDelegate {
+    func textViewDidChange(_ textView: UITextView) {
+        self.placeHolderLabel.textColor = introductionTextView.text.count == 0 ? .lightGray : .clear
+    }
+}
+
+// MARK: - KeyboardControl
+
+extension AddGatheringViewController {
+    private func getKeyboardNotification() {
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow(_:)), name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide(_:)), name: UIResponder.keyboardWillHideNotification, object: nil)
```

Comment:
p1;
NotificationCenter에 addObserver한 거는 직접 해제해주지 않으면 메모리 릭이 발생해요.
```
    deinit {
        NotificationCenter.default.removeObserver(self)
    }
```
뷰컨에서 옵저버 해제코드 추가해주세요

Reply:
수정했습니다! 5f1045049b245b004b2eaf058294d59067a488ed
[removeObserver(_:) 공식 문서](https://developer.apple.com/documentation/foundation/notificationcenter/1413994-removeobserver)의 Discussion 내용에 따라 추가한 각 옵저버의 이름을 지정해 해제했습니다. 
"When removing an observer, remove it with the most specific detail possible. For example, if you used a name and object to register the observer, use [removeObserver(_:name:object:)](doc://com.apple.documentation/documentation/foundation/notificationcenter/1407263-removeobserver?language=swift) with the name and object."

추후 시간이 되면 옵저버를 추가할 때 참고한 블로그의 다른 글([iOS의 Notification Center 확장, 그리고 Memory Leak 피하기](https://seizze.github.io/2019/11/28/iOS의-Notification-Center-확장,-그리고-Retain-Cycle-피하기.html))을 참고해 NotificationCenter에서 가져오는 정보를 줄이고, 자동으로 removeObserver가 불리도록 반영해보겠습니다. 

PR 코멘트를 보실 분들을 위한 **추가정보: iOS 9 부터는 addObserver(_:selector:name:object:)는 꼭 해제해줄 필요는 없습니다.**
- 하지만 **[addObserver(forName:object:queue:using:)](doc://com.apple.documentation/documentation/foundation/notificationcenter/1411723-addobserver?language=swift)의 경우는 해제가 필요**해서 추후 헷갈릴 수 있고, 해제하지 않을 이유도 없으므로 해제하는 편이 좋은 것 같아요. 


자료
- 애플 공식 문서2개 ([removeObserver(_:)](https://developer.apple.com/documentation/foundation/notificationcenter/1413994-removeobserver), [addObserver(_:selector:name:object:)](https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver))의 Discussion에서 iOS 9.0 이후부터는 addObserver(_:selector:name:object:)의 경우 해제하지 않아도 된다고 합니다. 
"If your app targets iOS 9.0 and later or macOS 10.11 and later, and you used [addObserver(_:selector:name:object:)](doc://com.apple.documentation/documentation/foundation/notificationcenter/1415360-addobserver?language=swift), you do not need to unregister the observer. If you forget or are unable to remove the observer, the system cleans up the next time it would have posted to it."

Reply:
자세한 정보 감사합니다!! 덕분에 저도 많이 배워갑니당👍👍👍